src/core/theme/colors.ts

```diff
@@ -106,9 +106,13 @@ export const gradients = {
   whiteBaseNegative: `linear-gradient(-180deg, ${alphaHex(0.1)(
     colors.whiteBase,
   )} 0%, ${alphaHex(0)(colors.whiteBase)} 100%)`,
-  whiteBaseToDepthLight2: `linear-gradient(0deg, ${colors.depthLight2} 0%, ${colors.whiteBase} 100%)`,
+  whiteBaseToDepthLight2: `linear-gradient(-180deg, ${alphaHex(1)(
```

Comment:
Is the alphaHex necessary for the first color as it has no transparency?

Reply:
Doesn't seem to be. Removed it 👍